src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/MetricsFormattingTests.cs

```diff
@@ -44,14 +44,16 @@ public async Task HistogramFormat_Test()
         {
             List<ICounterPayload> payload = new();
 
-            payload.Add(new AggregatePercentilePayload(MeterName, InstrumentName, "DisplayName", string.Empty, string.Empty,
+            CounterMetadata counterInfo = new CounterMetadata(MeterName, InstrumentName, null, null, null);
```

Comment:
Are the null values for meterTags, instrumentTags, and scopeHash? Could you use named parameters to make it more clear as to what each of these are?